packages/core/src/node/bunyan-logger-server.ts

```diff
@@ -42,11 +42,12 @@ export class BunyanLoggerServer implements ILoggerServer {
         if (['trace', 'debug', 'info', 'warn', 'error', 'fatal'].indexOf(logLevel) < 0) {
             logLevel = 'info';
         }
+    }
 
-        this.loggers.push(bunyan.createLogger({
-            name: 'Theia',
-            level: logLevel
-        }));
+    create(serveroptions: object | undefined) {
```

Comment:
The idea was to make this configurable from the frontend like the childs...
I'm not sure if we really need to however...